### ISSUE TITLE
Add downwards pointing arrow

### ISSUE THREAD

**Issue by odin-** [webianks/PopupBubble#7]
Hangouts got a popup like bubble displaying "Jump to latest" at the bottom of the RecyclerView (scroll list). It got a down pointing arrow. So to be able to mimic that behaviour it would be nice if you could add a down pointing arrow. 

(I can also do a pull req if you want me to).

**Comment by webianks** [comment]
You can position bubble according to your need. You can set it to bottom from your layout and also you can change the arrow according to your choice look for same in redme, rest of the behaviour remains the same as on click of that button recyclerview will be scrolled to top.

**Comment by webianks** [comment]
And I don't see any "Jump to latest" button in hangouts. Please provide screenshot.

**Comment by odin-** [comment]
Yup I've added my own down pointing arrow. But to mimic Hangouts out of the box it might be nice if you included it default in your library.
![c3izajzwmaqptru](https://cloud.githubusercontent.com/assets/7396319/25324795/14b4ac42-28c8-11e7-9957-aa9a39de4bde.jpg)

**Comment by webianks** [comment]
Okay cool I got that. I will add this behaviour.

**Comment by odin-** [comment]
In that case I guess this code in onTouchEvent is problematic since it auto scrolls to position 0. I would like to scroll to last position:

                    if (recyclerView != null)
                         recyclerView.smoothScrollToPosition(0);

**Comment by webianks** [comment]
I will need to this also to check the behaviour of recyclerview. I am sure that in the chat section you have reverse layout meaning last element becomes first element and then the above code should not raise any issue but it needs to be tested.
